### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by edunham**
The README stuff could be wrong, but it details the steps I've taken so far to get http://54.176.36.178 looking like http://ncameron.org/perf-rustc/

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+rustc-perf
+----------
+
+This is the website for Rust compiler performance monitoring. 
+
+https://github.com/nrc/benchmarks contains the raw data from which rustc-perf
```
Comment (nrc) [inline README.md]: SHoudl be https://github.com/nrc/rustc-timing

File: README.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+
+```
+sudo apt-get install npm apache2 nodejs
+git clone https://github.com/nrc/rustc-perf.git /var/www/html/
+cd /var/www/html/rustc-perf; ./init.sh
+```
+
+The following lines in `/etc/apache2/sties-enabled/000-default.conf` proxy
+traffic to the app's custom port. This is historical cruft from its previous
+shared setup.
```
Comment (nrc) [inline README.md]: It's not clear if this is historical cruft that is needed anyway or should be deleted by the user

File: init.sh [hunk 3]
```diff
@@ -0,0 +1,2 @@
+sudo npm install nodegit
+git clone https://github.com/nick29581/rustc-timing.git data
```
Comment (nrc) [inline init.sh]: s/nick29581/nrc

File: README.md (hunk 1 above)
Comment (edunham) [inline README.md]: oops, that was a pastefail. Thanks for catching it!

File: README.md (hunk 2 above)
Comment (edunham) [inline README.md]: I didn't know that either when writing the commit. Pretty sure it's actually essential for the "both halves of frontend share one host" setup. Rewritten.

**Pull request merged**